gallery/src/styles/NxList/NxListMultiLineExample.html

```diff
@@ -6,27 +6,32 @@
     distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
 
 -->
-<div class="nx-list">
-  <h4 class="nx-list__title">
-    List title
-  </h4>
-  <ul class="nx-list">
-    <li class="nx-list__item">
-      Item 1
-      <p class="nx-list__subtext">This is a second line.</p>
-    </li>
-    <li class="nx-list__item">
-      Item 2
-      <p class="nx-list__subtext">
-        This is a second line. It includes text that might relate to the top line or might not.
-      </p>
-    </li>
-    <li class="nx-list__item">
-      Item 3
-      <p class="nx-list__subtext">
-        A long line of subtext that wraps. Add <code class="nx-code">.nx-truncate-ellipsis</code>
-        to the <code class="nx-code">&lt;p&gt;</code> tag if you don't want the text to wrap.
-      </p>
-    </li>
-  </ul>
-</div>
+<ul class="nx-list">
+  <li class="nx-list__item">
+    <span class="nx-list__text">
+      Text wrapping - text that is very long and will wrap onto the next line, standard list item text that is very
+      long and will wrap onto the next line
+    </span>
+  </li>
+  <li class="nx-list__item">
+    <span class="nx-list__text nx-truncate-ellipsis">
+      Text truncation - Add <code class="nx-code">.nx-truncate-ellipsis</code> to
+      <code class="nx-code">.nx-list__item</code> to cause the text to truncate at the right edge of the list item.
+    </span>
+  </li>
+  <li class="nx-list__item">
+    <span class="nx-list__text">Subtext wrapping</span>
+    <p class="nx-list__subtext">
+      A long line of subtext that wraps. Add <code class="nx-code">.nx-truncate-ellipsis</code>
+      to the <code class="nx-code">&lt;p&gt;</code> tag if you don't want the text to wrap. More text to trigger the
+      wrapping
+    </p>
+  </li>
+  <li class="nx-list__item">
+    <span class="nx-list__text">Subtext truncation</span>
+    <p class="nx-list__subtext nx-truncate-ellipsis">
+      Add <code class="nx-code">.nx-truncate-ellipsis</code> to <code class="nx-code">.nx-list__subtext</code> to
+      cause the text to truncate at the right edge of the list item.
```

Comment:
We'll probably want to know how the action buttons and chevron interact with the wrapping and truncation, should more examples be made for those cases?

Reply:
> We'll probably want to know how the action buttons and chevron interact with the wrapping and truncation, should more examples be made for those cases?

https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/344242f37afb43e9271a20254a4020d4726dfb66

Reply:
> > We'll probably want to know how the action buttons and chevron interact with the wrapping and truncation, should more examples be made for those cases?
>
> 344242f

Maybe also make the "This is some sub-text to demonstrate spacing" text longer to show how subtext and buttons interact.

Reply:
> > > We'll probably want to know how the action buttons and chevron interact with the wrapping and truncation, should more examples be made for those cases?
> > 
> > 
> > [344242f](https://github.com/sonatype/sonatype-react-shared-components/commit/344242f37afb43e9271a20254a4020d4726dfb66)
> 
> Maybe also make the "This is some sub-text to demonstrate spacing" text longer to show how subtext and buttons interact.

Demonstrated wrapping and truncation: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/d0aa9e034c9729ef62cfded7e5c1deaccf1a54c0